script.js

```diff
@@ -2,57 +2,88 @@ if (localStorage.getItem('Added Books') == null) {
   localStorage.setItem('Added Books', JSON.stringify([]));
 }
 
-let storedBooksList = JSON.parse(localStorage.getItem('Added Books'));
+class BooksCollection {
+  constructor() {
+    this.books = JSON.parse(localStorage.getItem('Added Books'));
+  }
 
-function updatelocalStorage() {
-  localStorage.setItem('Added Books', JSON.stringify(storedBooksList));
-}
+  updatelocalStorage() {
+    localStorage.setItem('Added Books', JSON.stringify(this.books));
+  }
 
-function createBooks(arr) {
-  let books = '';
-  for (let i = 0; i < arr.length; i += 1) {
-    arr[i].id = i;
-    books += `<p>${arr[i].title}</p>
-    <p>${arr[i].author}</p>
-    <button onclick="removeBook(${i})">Remove</button>
-    <hr/>`;
+  /* eslint-disable class-methods-use-this */
+  createBooks(arr) {
+    let books = '';
+    for (let i = 0; i < arr.length; i += 1) {
+      arr[i].id = i;
+      books += `<div class="book-line"><p>"${arr[i].title}"</p>
+      <p>${arr[i].author}</p>
+      <button class="remove" onclick="booksCollection.removeBook(${i})">Remove</button>
+      </div>`;
+    }
+    return books;
   }
-  return books;
-}
 
-function displayBooks() {
-  const container = document.querySelector('.books-list');
-  container.innerHTML = createBooks(storedBooksList);
-}
+  /* eslint-enable class-methods-use-this */
+  displayBooks() {
+    const container = document.querySelector('.books-list');
+    container.innerHTML = this.createBooks(this.books);
+  }
 
-function addNewBook(title, author) {
-  const id = storedBooksList.length;
-  const book = {
-    id,
-    title,
-    author,
-  };
-  storedBooksList.push(book);
-  updatelocalStorage();
-  displayBooks();
-  document.querySelector('.title').value = '';
-  document.querySelector('.author').value = '';
-}
+  addNewBook(title, author) {
+    const id = this.books.length;
+    const book = {
+      id,
+      title,
+      author,
+    };
+    this.books.push(book);
+    this.updatelocalStorage();
+    this.displayBooks();
+    document.querySelector('.title').value = '';
+    document.querySelector('.author').value = '';
+  }
 
-function removeBook(id) {
-  storedBooksList = storedBooksList.filter((el) => el.id !== id);
-  updatelocalStorage();
-  displayBooks();
+  removeBook(id) {
+    this.books = this.books.filter((el) => el.id !== id);
+    this.updatelocalStorage();
+    this.displayBooks();
+  }
+
```

Comment:
- [ ] Your code should have a class `methods` to `add` and `remove` books, kindly consider creating a class with that name (methods) and include the methods responsible for adding and removing the books to it

Reply:
I have already created class methods for adding and removing. This part of the code is not in my code